ansible_collections/arista/avd/molecule/cv_deploy/cc_false.yml

```diff
@@ -12,46 +12,50 @@
     structured_dir: "{{ playbook_dir }}/intended/structured_configs/test_configs"
     intended_tag_device: avd-ci-leaf1
     intended_tags: "{{ lookup('file', structured_dir ~ '/' ~ intended_tag_device ~ '.yml')| from_yaml }}"
+    test_id: "cc-false"
```

Comment:
Should we also add tags with this name on all the tasks related to this test? To make it easier to just run some of them?